cfme/intelligence/reports/reports.py

```diff
@@ -556,6 +593,20 @@ def step(self):
         self.view.report_info.select()
 
 
+@navigator.register(Report, "ScheduleReport")
+class ReportSchedule(CFMENavigateStep):
+    VIEW = NewScheduleView
+    prerequisite = NavigateToAttribute("appliance.server", "CloudIntelReports")
+
+    def step(self):
+        self.prerequisite_view.reports.tree.click_path(
+            "All Reports",
+            self.obj.type or self.obj.company_name,
```

Comment:
I am suspect about these or. are those required?

Reply:
It is required to navigate to the details page of the report and then select `Add a new Schedule` from the configuration menu.